src/EnergyPlus/UnitarySystem.cc

```diff
@@ -14838,9 +14857,14 @@ namespace UnitarySystems {
         } else {
 
             DataLoopNode::Node(AirControlNode).MassFlowRate = airMdot;
-            thisSys.FanPartLoadRatio =
-                max(0.0, ((airMdot - (systemMaxAirFlowRate * lowSpeedRatio)) / ((1.0 - lowSpeedRatio) * systemMaxAirFlowRate)));
-
+            if ( lowSpeedRatio != 1.0 ) {
+                // division by zero when lowSpeedRatio == 1.0
+                thisSys.FanPartLoadRatio =
+                    max( 0.0, ((airMdot - (systemMaxAirFlowRate * lowSpeedRatio)) / ((1.0 - lowSpeedRatio) * systemMaxAirFlowRate)) );
+            }
+            else {
+                thisSys.FanPartLoadRatio = 1.0;
+            }
```

Comment:
Running the defect file after the above changes caused division by zero. Thus, this logic was added.